doc/api/http.md

```diff
@@ -1443,16 +1443,31 @@ This event is guaranteed to be passed an instance of the {net.Socket} class,
 a subclass of {stream.Duplex}, unless the user specifies a socket
 type other than {net.Socket}.
 
-### `server.close([callback])`
+### `server.close([options][, callback])`
 
 <!-- YAML
 added: v0.1.90
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/42812
+    description: The `options` argument is now supported.
 -->
 
+* `options` {Object}
+  * `force` {string} Specified which kind of already opened connections to
+    forcefully close. Supported values are:
+    * `keep-alive`: Will close all idle connections (which are using
+      HTTP Keep-Alive).
```

Comment:
No, it's not like that.

`keep-alive` will purge all the connections which are not sending,receiving or waiting for data and are opened just because of HTTP Keep-Alive.